pkgs/development/python-modules/raylib-python-cffi/default.nix

```diff
@@ -0,0 +1,56 @@
+{
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  cffi,
+  pkg-config,
+  glfw,
+  libffi,
+  raylib,
+  physac,
+  raygui,
+  lib
+}:
+
+buildPythonPackage rec {
+  pname = "raylib-python-cffi";
+  version = "5.0.0.2";
+
+  src = fetchFromGitHub {
+    owner = "electronstudio";
+    repo = "raylib-python-cffi";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-DlnZRJZ0ZnkLii09grA/lGsJHPUYrbaJ55BVWJ8JzfM=";
+  };
+
+  build-system = [
+    setuptools
+    cffi
```

Comment:
`cffi` should go to `dependencies`.
https://github.com/electronstudio/raylib-python-cffi/blob/1232b700a530b2bc7080a4177714aebd9b72a419/setup.py#L41